docs/tutorials/installation.md

```diff
@@ -1,10 +1,319 @@
 (installation)=
 
-# Installation
+# How to install JAX
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+This guide provides instructions for:
 
-For the time being, you may find some related content in the old documentation:
-- {doc}`../installation`
+- Installing JAX binary packages for supported platforms using `pip` or `conda`
+- Using [NVIDIA GPU Docker containers](#docker-containers-nvidia-gpu)
+- [Building JAX from source](developer.md#building-from-source)
+
+**TL;DR** For most users, a typical JAX installation may look something like this:
+
+```
+# CPU-only for Linux, macOS or Windows users:
+$ pip install "jax[cpu]"
+
+# NVIDIA GPU with CUDA 12 for Linux users:
+$ pip install "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+
+# Apple ARM-based GPU (macOS users):
+# Follow Apple's instructions at https://developer.apple.com/metal/jax/
+```
+
+Continue reading this guide to learn about more options.
+
+## Supported platforms
+
+In the table below, check if your setup is supported. And if it says _"yes"_ or _"experimental"_ you can click on the corresponding link to learn how to install JAX in greater detail.
+
+|                  | Linux, x86_64            | Linux, aarch64     | macOS, Intel x86_64        | macOS, ARM-based           | Windows, x86_64 | Windows WSL2, x86_64        |
+|------------------|--------------------------|--------------------|----------------------------|----------------------------|-----------------|-----------------------------|
+| CPU              | [yes](#cpu)              | [yes](#cpu)        | [yes](#cpu)                | [yes](#cpu)                | [yes](#cpu)     | [yes](#cpu)                 |
+| NVIDIA GPU       | [yes](#nvidia-gpu)       | [yes](#nvidia-gpu) | no                         | n/a                        | no              | [experimental](#nvidia-gpu) |
+| Google Cloud TPU | [yes](#google-tpu)       | n/a                | n/a                        | n/a                        | n/a             | n/a                         |
+| AMD GPU          | [experimental](#amd-gpu) | no                 | no                         | n/a                        | no              | no                          |
+| Apple GPU        | n/a                      | no                 | [experimental](#apple-gpu) | [experimental](#apple-gpu) | n/a             |  n/a                        |
+
```

Comment:
We could think about replacing links in the table with sphinx cross-references as well, but that might take a bit of work. Probably fine to leave it for later.